src/collection/App.tsx

```diff
@@ -14,14 +14,15 @@ import IntroFacade from "./pages/facade/IntroFacade";
 import path from "path";
 
 
+const baseUrl = get("UseRemoteHeadless") ? REMOTE_HEADLESS_URL : LOCAL_SERVER_URL;
```

Comment:
How about abstract `REMOTE_HEADLESS_URL` to `HEADLESS_URL` and move this ternary operator to `config` module?